src/components/social/_snapchat.less

```diff
@@ -0,0 +1,11 @@
+.social-btn {
+  &.snapchat-ghost {
```

Comment:
can you remove the ghost in the class name just for naming conventions

Reply:
Alright !

Reply:
Alright !